scripts/release.py

```diff
@@ -9,15 +10,15 @@ def create_tag():
     # create new tag as per current release
     stream = os.popen('git describe --abbrev=0 --tags')
     current_tag = stream.read().strip()
-    # TODO(Xueyan): the current parsing is buggy, if the main version number is a two-digit number.
-    #      You need parse the string, e.g., the first string is 'v', and after 'v', there should be
-    #      two numbers seperated by a '.'
-    if current_tag[0] != 'v':
-        raise Exception("The tag name should start with v")
+
+    if not re.match("^v[0-9][.][0-9]", current_tag):
+        raise Exception("Error: tag name does not follow expected pattern.")
+
     major_version = int(current_tag[1])
```

Comment:
This only works with single-digit number.

Reply:
To address this, in you first get the index of the `.`.

Then get the major version by slicing the string from the second character until the `.` character.

For the minor version, just the characters after `.`.